auxiliary.ps1

```diff
@@ -0,0 +1,104 @@
+$FILTERS = "https://go.microsoft.com/fwlink/?linkid=875139"
+$STUDIOCOMPUTERNAME = "STUDIO"
+$CLIENTCOMPUTERNAME = "CL"
+$KITTYPE = "HLK"

```

Comment:
Should be configurable. I am missing the place where it can be changed.

Reply:
I think to use the same method as with `auto.bat`, just edit this file. What do you think about this?

Reply:
Edit by auto_hck?

Reply:
yes